runtime/compiler/z/codegen/J9UnresolvedDataReadOnlySnippet.cpp

```diff
@@ -122,12 +123,11 @@ J9::Z::UnresolvedDataReadOnlySnippet::emitSnippetBody()
       // Branch to doneLabel (BRCL 0xF,[doneLabel])
       *reinterpret_cast<int16_t*>(cursor) = 0xC0F4;
 
-      addressA = reinterpret_cast<intptr_t>(doneLabel->getCodeLocation());
-      addressB = reinterpret_cast<intptr_t>(cursor);
+		intptr_t doneLabelAddress = reinterpret_cast<intptr_t>(doneLabel->getCodeLocation());
```

Comment:
Tabs should be spaces here for consistency.

Reply:
Fixed in https://github.com/eclipse/openj9/compare/ee40f0a3614cab3847b46a4b61c6fb2795d4b775..3ac390dbe99682be85f71980c4568d19c47c35dd